cmd/activator/request_log.go

```diff
@@ -29,11 +30,20 @@ import (
 
 func updateRequestLogFromConfigMap(logger *zap.SugaredLogger, h *pkghttp.RequestLogHandler) func(configMap *corev1.ConfigMap) {
```

Comment:
this seems like it'll need updates to `request_log_test.go`

Reply:
Actually it was `queue_test.go`, for 0.17 we want things to work as previously so unit tests shouldnt break.

Reply:
`queue_test.go` failed because of the changes to `queue.go`. This requires changes to `request_log_test.go` because it changes the behaviour so should probably at least result in a new row in the table test or something :-).

Reply:
Yes I was refering to the failures, to be 100% ok we need to pass all cases via observability config logic.

Reply:
ah right, gotcha. I was just meaning "we probably need to add a test for the new logic here"